src/host_toolchains.py

```diff
@@ -140,7 +140,7 @@ def CMakeLauncherFlags():
 
 
 def NinjaJobs():
-  if UsingGoma():
+  if UsingGoma() and force_host_clang:
```

Comment:
I'm definitely missing context here. It seems to me that `UsingGoma()` and `force_host_clang` should be contradictory.

Reply:
It's actually the other way around. The "forced" clang is the one that comes from the chrome infra, so it's the one that's guaranteed to be in goma. (The "unforced" clang is just whatever the default local compiler is).